projects/storefrontapp-e2e-cypress/cypress/helpers/added-to-cart-modal.ts

```diff
@@ -0,0 +1,219 @@
+export const productId = '3595723';
+export const productId2 = '3325048';
+
+export function basicBehavior() {
+  // Type 1000 in the input to see if the value will change to maximum 'max stock'
+  cy.get('cx-product-summary .cx-item-counter__value')
+    .type('{selectall}{backspace}')
+    .type('1000')
+    .should('have.value', '22');
+
+  // check if the '+' button is disabled when the quantity is the maximum 'max stock'
+  cy.get('cx-product-summary .cx-item-counter__action')
+    .contains('+')
+    .should('be.disabled');
+
+  // Type 0 in the input to see if the value will change to minimum '1'
+  cy.get('cx-product-summary .cx-item-counter__value')
+    .type('{selectall}{backspace}')
+    .type('0')
+    .should('have.value', '1');
+
+  // check if the '-' button is disabled when the quantity is the minimum '1'
+  cy.get('cx-product-summary .cx-item-counter__action')
+    .contains('-')
+    .should('be.disabled');
+
+  // increase the quantity to 2 and add it to cart
+  cy.get('cx-product-summary .cx-item-counter__action')
+    .contains('+')
+    .click();
+
+  cy.get('cx-product-summary cx-add-to-cart button').click();
+
+  // Check if the text in the cart dialog and product summary in product details matches
+  cy.get('cx-added-to-cart-dialog .cx-name').then($cartItem => {
+    cy.get('cx-product-summary .name').then($productItem => {
+      expect($cartItem.text()).equal($productItem.text());
+    });
+  });
+
+  // quantity is correctly updated
+  cy.get('cx-added-to-cart-dialog .cx-quantity .cx-item-counter__value').should(
+    'have.value',
+    '2'
+  );
+  cy.get('cx-added-to-cart-dialog .cx-dialog-total').should(
+    'contain',
+    '2 items'
+  );
+
+  // check if the total is correct
+  cy.get('cx-added-to-cart-dialog .cx-total .cx-value').then(
+    $cartTotalItemPrice => {
+      const totalPrice = $cartTotalItemPrice.text().trim();
+      expect(totalPrice).equal('$4,004.96');
```

Comment:
I would extract the product's unit price, and then do an assertion on (2*unit price), instead of hardcoding the value here, in case those data changes, it makes our tests more fragile.